content/tbd/STRP.md

```diff
@@ -0,0 +1,38 @@
+---
+created: 2023-12-07
+contributors: Matthew McCarthy
+title: STRP
+url: https://strp.nl/
+locations:
+  - Eindhoven, NL
+start: "2006"
```

Comment:
is this put in as a number or a string (text) in your file? the property type should be a number